common/app-starters-security-common/src/main/java/org/springframework/cloud/stream/app/security/common/SecurityCommonAutoConfiguration.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.cloud.stream.app.security.common;
+
+import org.springframework.boot.autoconfigure.AutoConfigureBefore;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.builders.WebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+
+/**
+ * @author Christian Tzolov
+ * @author Artem Bilan
+ */
+@Configuration
+@AutoConfigureBefore(name = SecurityCommonAutoConfiguration.MANAGEMENT_WEB_SECURITY_AUTO_CONFIGURATION_CLASS)
+public class SecurityCommonAutoConfiguration {
+
+	public final static String MANAGEMENT_WEB_SECURITY_AUTO_CONFIGURATION_CLASS =
+			"org.springframework.boot.actuate.autoconfigure.security.servlet.ManagementWebSecurityAutoConfiguration";
+
+	/**
+	 * The custom {@link WebSecurityConfigurerAdapter} to disable security in the application
+	 * if {@code spring.cloud.security.enabled = false}.
+	 * When {@code spring.cloud.security.enabled = true} (default) then this configuration falls back to the default
+	 * Spring Security configuration.
+	 * @see org.springframework.boot.autoconfigure.security.servlet.SpringBootWebSecurityConfiguration
```

Comment:
And also `@see org.springframework.boot.actuate.autoconfigure.security.servlet.ManagementWebSecurityAutoConfiguration` as well

Reply:
The ManagementWebSecurityAutoConfiguration class is not provided in the module. Is it ok to have it in the javadoc?